### DISCUSSION TITLE
Can I import Statoil format data for simulation calculations？

### DISCUSSION THREAD

**Discussion by weishiyu1** [PMEAL/OpenPNM#2279] [Q&A]
Can I import Statoil format data for simulation calculations? Why do I make mistakes when using the example?

**Comment by jgostick** [comment]
Can you explain the error you're getting, as well as which verion of openpnm you are using?

**Reply by weishiyu1** [reply]
> Reference in ne

I want study the change of relative permebibity of fluid during the imbibiton process, and Mixed (Invasion) Percolation is a good example, however, i do not know how to simulate this process with Statoil format data,could you help me?

**Reply by weishiyu1** [reply]
> Can you explain the error you're getting, as well as which verion of openpnm you are using?

KeyError: 'pore.surface'

**Reply by jgostick** [reply]
So the network is looking for labels, such as pore.surface.  You can read more about labels [here](https://github.com/PMEAL/OpenPNM/blob/dev/examples/tutorials/using_and_creating_labels.ipynb).  I am not sure at which point the label is being looked up, but I assume it's after you've imported it into openpnm, and not during the import step?
